app/components/Form/PayForm.scss

```diff
@@ -172,4 +172,10 @@
       cursor: pointer;
     }
   }
+}
+
+input[type=number]{
```

Comment:
sorry super small but can you add a space between `input[type=number]` and `{`?

Reply:
Sure thing, I'll update it shortly.